src/routes/map/+page.svelte

```diff
@@ -98,45 +89,25 @@
 			<div class="flex grow flex-col">
 				<div class="relative h-full w-full">
 					<div
-						class="absolute left-0 top-0 z-[500] min-h-[4.5rem] w-64 rounded-br-md bg-white p-2 shadow"
+						class="absolute left-0 top-0 z-[500] flex w-fit flex-row items-center gap-1 rounded-br-md bg-white p-2 shadow"
 					>
-						{#if hoveredFeature && hoveredFeature.properties}
-							<div>
-								{hoveredFeature.properties.NAME}
-							</div>
-							<div>
-								{#if hoveredMetric}
-									{hoveredMetric.value.toFixed(0)} tonnes CO₂e per capita
-								{:else}
-									No data available
-								{/if}
-							</div>
-						{:else}
-							<div>Click on a country or</div>
-							<details class="dropdown">
-								<summary class="btn btn-ghost btn-sm w-60 font-normal"
-									>Select country &#9660;</summary
-								>
-								<!-- TODO dont hardcode height and width -->
-								<div
-									class="card dropdown-content compact rounded-box z-[500] h-[600px] w-[900px] overflow-y-scroll bg-base-100 shadow"
-								>
-									<!-- TODO add filter input box to make it easier to find country -->
-									<div class="card-body">
-										<RegionList regions={data.regions} />
-									</div>
+						<div>Click on a country or</div>
+						<details class="dropdown">
+							<summary class="btn btn-ghost btn-sm font-normal">Select country &#9660;</summary>
```

Comment:
I think the button looks better (and better aligned) with 

```suggestion
							<summary class="btn btn-ghost btn-sm normal-case">select country &#9660;</summary>
```

Reply:
So you want the button to look like the text before it.

Reply:
Hmm it looks like my suggestions don't display correctly, at least not on my phone. I can try to see if I still have my edits somewhere. But yes, I want it to look like on the screenshot.